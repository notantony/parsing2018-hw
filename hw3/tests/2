mean l r =
= m / + l r 2