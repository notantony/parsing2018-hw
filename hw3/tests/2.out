void mean(int l, int r) {
    int m;
    m = ((l + r) / 2);
}
